opencensus/common/monitored_resource/k8s_utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from opencensus.common.monitored_resource import gcp_metadata_config
+
+# Env var that signals that we're in a kubernetes container
+_KUBERNETES_SERVICE_HOST = 'KUBERNETES_SERVICE_HOST'
+
+# Name of the cluster the container is running in
+CLUSTER_NAME_KEY = 'k8s.io/cluster/name'
+
+# ID of the instance the container is running on
+NAMESPACE_NAME_KEY = 'k8s.io/namespace/name'
+
+# Container pod ID
+POD_NAME_KEY = 'k8s.io/pod/name'
+
+# Container name
+CONTAINER_NAME_KEY = 'k8s.io/container'
```

Comment:
This should be 'k8s.io/container/name': https://github.com/census-instrumentation/opencensus-specs/pull/235/files#diff-b965ed607d4cefb5c41b99223e8e0ac0R16.